### DISCUSSION TITLE
Network Policy not working as expected.

### DISCUSSION THREAD

**Discussion by erchauhannitin** [kubernetes/website#42070] [Advice on using this repository]
https://kubernetes.io/docs/tasks/administer-cluster/declare-network-policy/#test-the-service-by-accessing-it-from-another-pod

Followed the steps on the above page but output is not matching the expectations. 
In both cases access is there. It is not blocked. wget command is able to access in both cases.

**Comment by divya-mohan0209** [comment]
Hello @erchauhannitin : Please do not open discussion items for support requests. Please either use one of the forums and/or use the issues section for further support
